src/harness/unittests/typingsInstaller.ts

```diff
@@ -935,25 +935,25 @@ namespace ts.projectSystem {
                 import * as cmd from "commander
                 `
             };
-            session.executeCommand(<server.protocol.OpenRequest>{

```

Comment:
The casts in this file would be unneeded by using the overloaded signature, too.